spec.bs

```diff
@@ -875,19 +875,14 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
-1. If |config|["{{AuctionAdConfig/directFromSellerSignals}}"] [=map/exists=]:
-  1. TODO: The receiving end of this isn't specified yet, so there is no place to put the computed value.
-  1. [=Handle an input promise in configuration=] given |auctionConfig| and |config|["{{AuctionAdConfig/directFromSellerSignals}}"]:
+1. If |config|["{{AuctionAdConfig/directFromSellerSignalsHeaderAdSlot}}"] [=map/exists=]:
+  1. [=Handle an input promise in configuration=] given |auctionConfig| and |config|["{{AuctionAdConfig/directFromSellerSignalsHeaderAdSlot}}"]:
     * To parse the value |result|:
-      1. Let |directFromSellerSignalsPrefix| be the result of running the [=URL parser=] on |result|.
-      1. [=exception/Throw=] a {{TypeError}} if any of the following conditions hold:
-        * |directFromSellerSignalsPrefix| is failure;
-        * |directFromSellerSignalsPrefix| is not [=same origin=] with |auctionConfig|'s
-          [=auction config/seller=];
-        * |directFromSellerSignalsPrefix|'s [=url/query=] is not null.
-      1. [=Assert=]: |directFromSellerSignalsPrefix|'s [=url/scheme=] is "`https`".
+      1. Set |auctionConfig|'s [=auction config/direct from seller signals header ad slot=] to |result|.
+      1. Set |auctionConfig|'s [=auction config/expects direct from seller signals header ad slot=] to true.

```

Comment:
Related to https://github.com/WICG/turtledove/issues/788, it seems weird that we're setting individual members to failure but never setting the config itself to failure. Maybe that's a mistake? Probably fine to move forward with this PR for now though since it is consistent, and all of that will [be fixed soon](https://github.com/WICG/turtledove/issues/788#issuecomment-1712117723).

Reply:
yeah, also saw the problem when looking at #788 
Will fix this in that fix PR as well.